pkgs/applications/audio/lx-music-desktop/default.nix

```diff
@@ -0,0 +1,103 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, dpkg
+, alsa-lib
+, at-spi2-atk
+, cups
+, glib
+, gtk3
+, libGL
+, libdrm
+, mesa
+, nss
+, systemd
+, xorg
+}:
+
+let
+  pname = "lx-music-desktop";
+  version = "2.6.0";
+
+  buildUrl = version: arch: "https://github.com/lyswhut/lx-music-desktop/releases/download/v${version}/lx-music-desktop_${version}_${arch}.deb";
```

Comment:
version can be removed from argument, version from let block can be used instead